config/locales/en.yml

```diff
@@ -907,122 +907,122 @@ en:
-          suction_point: "Emergency Suction Point"
-          water_tank: "Emergency Water Tank"
+          siren: "Siren"
+          suction_point: "Water Drafting Site"
+          water_tank: "Emergency Water Reservoir"
         highway:
           abandoned: "Abandoned Highway"
-          bridleway: "Bridleway"
-          bus_guideway: "Guided Bus Lane"
+          bridleway: "Bridle Path"
+          bus_guideway: "Guided Busway"
           bus_stop: "Bus Stop"
-          construction: "Highway under Construction"
-          corridor: "Corridor"
+          construction: "Highway Under Construction"
+          corridor: "Indoor Corridor"
           crossing: "Crossing"
-          cycleway: "Cycle Path"
+          cycleway: "Bicycle Path"
           elevator: "Elevator"
           emergency_access_point: "Emergency Access Point"
-          emergency_bay: "Emergency Bay"
+          emergency_bay: "Emergency Stopping Place"
           footway: "Footpath"
           ford: "Ford"
-          give_way: "Give Way Sign"
+          give_way: "Yield Sign"
           living_street: "Living Street"
           milestone: "Milestone"
-          motorway: "Motorway"
-          motorway_junction: "Motorway Junction"
-          motorway_link: "Motorway Road"
+          motorway: "Controlled-Access Highway"
+          motorway_junction: "Highway Exit"
+          motorway_link: "Highway Ramp"
           passing_place: "Passing Place"
           path: "Path"
-          pedestrian: "Pedestrian Way"
-          platform: "Platform"
+          pedestrian: "Pedestrian Mall"
+          platform: "Transit Platform"
           primary: "Primary Road"
-          primary_link: "Primary Road"
+          primary_link: "Primary Link Road"
           proposed: "Proposed Road"
-          raceway: "Raceway"
+          raceway: "Motor Racetrack"
           residential: "Residential Road"
           rest_area: "Rest Area"
           road: "Road"
           secondary: "Secondary Road"
-          secondary_link: "Secondary Road"
+          secondary_link: "Secondary Link Road"
           service: "Service Road"
-          services: "Motorway Services"
+          services: "Service Area"
           speed_camera: "Speed Camera"
-          steps: "Steps"
+          steps: "Stairway"
           stop: "Stop Sign"
           street_lamp: "Street Lamp"
           tertiary: "Tertiary Road"
-          tertiary_link: "Tertiary Road"
-          track: "Track"
+          tertiary_link: "Tertiary Link Road"
+          track: "Land Management Road or Vehicular Trail"
           traffic_mirror: "Traffic Mirror"
           traffic_signals: "Traffic Signals"
           trailhead: "Trailhead"
           trunk: "Trunk Road"
-          trunk_link: "Trunk Road"
-          turning_circle: "Turning Circle"
-          turning_loop: "Turning Loop"
-          unclassified: "Unclassified Road"
+          trunk_link: "Trunk Link Road"
+          turning_circle: "Cul-de-Sac"
+          turning_loop: "Cul-de-Sac"
```

Comment:
“Cul-de-sac” can refer to either the whole street or just its bulbous end. “Cul-de-sac street” emphasizes the broader meaning. If the end of the street has a turning hammer, we’d tag it as a `highway=turning_circle` and I think most people would handwave about it being a cul-de-sac. But if it ends abruptly without a widening, it would be a “dead end”, not a “cul-de-sac”.